how-to/template-how-to.md

```diff
@@ -36,7 +36,7 @@ Lead-in sentence for an ordered list:
 
 ### Step 3: Optional: title for step - code snippet
 
-Lead in sentence explaining the code snippet. E.g.: 
+Lead-in sentence explaining the code snippet. E.g.: 
```

Comment:
I know it is not part of the core issue here, but `E.g.:` is not recommended according to https://developers.google.com/style/abbreviations#dont-use

Could you please change this to:

"For example: "

Reply:
Replaced abbreviation by its full-form. Glad that I could help @jaredmorgs :)